{"uid":"8ee1764bcaf424d7","name":"test_github","fullName":"tests.test_selene#test_github","historyId":"5167f7c71535600a353c42dca9c39857","time":{"start":1697388832574,"stop":1697388832574,"duration":0},"status":"skipped","statusMessage":"Skipped: unconditional skip","statusTrace":"('/home/runner/work/qaguru_lesson_9/qaguru_lesson_9/tests/test_selene.py', 8, 'Skipped: unconditional skip')","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"configure_browser","time":{"start":1697388823653,"stop":1697388823653,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0}],"afterStages":[{"name":"configure_browser::0","time":{"start":1697388832577,"stop":1697388833555,"duration":978},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 118\nCurrent browser version is 117.0.5938.132 with binary path /opt/google/chrome/chrome\nStacktrace:\n#0 0x55de7c6d3fb3 <unknown>\n#1 0x55de7c3a74a7 <unknown>\n#2 0x55de7c3df95b <unknown>\n#3 0x55de7c3dec1a <unknown>\n#4 0x55de7c3d9acf <unknown>\n#5 0x55de7c3d710c <unknown>\n#6 0x55de7c419ac6 <unknown>\n#7 0x55de7c410713 <unknown>\n#8 0x55de7c3e318b <unknown>\n#9 0x55de7c3e3f7e <unknown>\n#10 0x55de7c6998d8 <unknown>\n#11 0x55de7c69d800 <unknown>\n#12 0x55de7c6a7cfc <unknown>\n#13 0x55de7c69e418 <unknown>\n#14 0x55de7c66b42f <unknown>\n#15 0x55de7c6c24e8 <unknown>\n#16 0x55de7c6c26b4 <unknown>\n#17 0x55de7c6d3143 <unknown>\n#18 0x7f97fc294b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/qaguru_lesson_9/qaguru_lesson_9/tests/conftest.py\", line 16, in configure_browser\n    browser.close()\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selene/core/entity.py\", line 1209, in close\n    self.driver.close()\n    ^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selene/core/entity.py\", line 1100, in driver\n    return self.config.driver\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selene/core/configuration.py\", line 343, in __get__\n    driver = config.build_driver_strategy(config)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selene/core/configuration.py\", line 171, in _build_local_driver_by_name_or_remote_by_url_and_options\n    return {  # type: ignore\n           ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selene/core/configuration.py\", line 93, in install_and_build_chrome\n    return Chrome(\n           ^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 56, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 205, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 289, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0}],"labels":[{"name":"tag","value":"@pytest.mark.skip"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_selene"},{"name":"host","value":"fv-az341-644"},{"name":"thread","value":"2202-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_selene"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.skip"]},"source":"8ee1764bcaf424d7.json","parameterValues":[]}